sass/buttons.scss

```diff
@@ -81,7 +81,7 @@ a.read-more:active {
     padding: 3px 8px;
     margin-bottom: 5px;
     text-align: center;
-    font-size: 13px;
+    font-size: 0.8125rem;
```

Comment:
Most of the theme uses `rem` font sizes, but there were a few spots that using `px` for no discernible reason. I updated them to use`rem`.

Reply:
It's because I'm not a frontend developer. I forked the original theme and added some random changes on top :smile: 

I guess it makes sense to use the `rem` units everywhere :+1: